_docs/runtime/installation.md

```diff
@@ -1,10 +1,91 @@
 ---
-title: "Installation and configuration"
+title: "Runtime installation and configuration"
 description: ""
 group: runtime
 toc: true
 ---
 
-Coming soon
 
-Documentation on how to install it, what are the prerequisites for installing it and what are the different options for various Cloud providers
+Codefresh runtime installs Codefresh Software Development Platoform (CSDP), comprising Argo CD ecosystem components and Codefresh-specific components. The Argo CD components are derieved from a conformed fork.
```

Comment:
i tihnk we should say that it's our forked enterprise versions of argo echo systems, it's not the opensource exact components

Reply:
lets put link to architectuer article from here